src/solid/lib/base.ts

```diff
@@ -0,0 +1,122 @@
+import { createWatch, is, Scope, Store } from "effector";
+import { throwError } from "./throw";
+import {
+  Accessor,
+  createMemo,
+  createSignal,
+  onCleanup
+} from "solid-js";
+
+const stateReader = <T,>(store: Store<T>, scope?: Scope) =>
```

Comment:
```suggestion
const stateReader = <T>(store: Store<T>, scope?: Scope) =>
```

Do we really need `,` here?